### DISCUSSION TITLE
Adopting an existing device simply to restore

### DISCUSSION THREAD

**Discussion by matthew-lenz-otr** [jlesage/docker-crashplan-pro#481] [Q&A]
@jlesage is there any real difference in procedure (vs your instructions on taking over a device) when all I'm looking to do is restore all the data from a device that is no longer bootable?  Also I do still have a local crashplan drive with the same backup on it as well.

**Comment by skadann** [comment]
@matthew-lenz-otr did you perform a restore? Any advice?

**Reply by matthew-lenz-otr** [reply]
don't remember what I did unfortunately.
